### ISSUE TITLE
(4116,P4) R17 UX1 - Tab order in the registration page is inconsistent. Safari only.

### ISSUE THREAD

**Issue by rahul-kumar11** [subrata1106/EnrolmentRel17UAT#42]
The full registration page should have all the form fields accessible by tab order and in sequence.

This issue seems to only apply to Safari on Mac. Chrome, IE 11 and Firefox behaved correctly and consistently.

The tab order in Safari browser is the following:
1. Full name
2. Username
3. Password
4. Confirm password
5. Country
6. Security Question
7. Security answer
8. URL Address bar
9. Page language Drop-down box
10. (Back to Full name)
Steps To Reproduce	1. Go to https://testenrolment.oxfordlearn.com [^]
2. Click Register here
3. Click the full name input field
4. Press TAB
5. Note down the sequence
